README.md

```diff
@@ -9,7 +9,8 @@ The following steps will install [Tekton Pipeline](https://github.com/tektoncd/p
     
 2. Apply Operator CRD
 
-    `kubectl apply -f deploy/crds/*_crd.yaml`
+    `kubectl apply -f deploy/crds/operator_v1alpha1_config_crd.yaml`
```

Comment:
isn't the old version better? i.e. apply all (`*`) crd.yaml files?

Reply:
Will raise exception if the number of files is more than one

Reply:
🐼 learned something today 😀